DataManagementSystem/Client/ConsistencyInspector.py

```diff
@@ -439,17 +439,17 @@ def compareChecksum( self, lfns ):
       if not oneGoodReplica:
         if lfn in lfnNotExisting:
           gLogger.info( "=> All replicas are missing", lfn )
-          retDict['MissingAllReplicas'][ lfn] = 'All'
+          retDict['MissingAllReplicas'][lfn] = 'All'
         else:
           gLogger.info( "=> All replicas have bad checksum", lfn )
-          retDict['AllReplicasCorrupted'][ lfn ] = csDict[ lfn ]
+          retDict['AllReplicasCorrupted'][lfn] = csDict[ lfn ]
```

Comment:
so should there be spaces inside the brackets or not? Because there are still spaces inside the brackets on the right hand side.